--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -37,7 +37,7 @@ tests_rpath = no
 endif
 
 check_PROGRAMS = arextract arsymtest newfile saridx scnnames sectiondump \
-		  showptable update1 update2 update3 update4 test-nlist \
+		  showptable update1 update2 update3 update4 \
 		  show-die-info get-files next-files get-lines next-lines \
 		  get-pubnames \
 		  get-aranges allfcts line2addr addrscopes funcscopes \
