Index: elfutils-0.188/debuginfod/debuginfod-client.c
===================================================================
--- elfutils-0.188.orig/debuginfod/debuginfod-client.c	2022-11-13 15:20:33.953176394 -0500
+++ elfutils-0.188/debuginfod/debuginfod-client.c	2022-11-13 15:20:33.949176380 -0500
@@ -47,6 +47,10 @@
 #include <stdlib.h>
 #include <gelf.h>
 
+#ifndef PATH_MAX
+#define PATH_MAX 1024
+#endif
+
 /* We might be building a bootstrap dummy library, which is really simple. */
 #ifdef DUMMY_LIBDEBUGINFOD
 
Index: elfutils-0.188/debuginfod/debuginfod.cxx
===================================================================
--- elfutils-0.188.orig/debuginfod/debuginfod.cxx	2022-11-13 15:20:33.953176394 -0500
+++ elfutils-0.188/debuginfod/debuginfod.cxx	2022-11-13 15:20:33.953176394 -0500
@@ -32,6 +32,8 @@
   #include "config.h"
 #endif
 
+#include <argp.h>
+
 extern "C" {
 #include "printversion.h"
 #include "system.h"
@@ -40,7 +42,6 @@
 #include "debuginfod.h"
 #include <dwarf.h>
 
-#include <argp.h>
 #ifdef __GNUC__
 #undef __attribute__ /* glibc bug - rhbz 1763325 */
 #endif
@@ -640,10 +641,10 @@
       scan_source_info = false;
       break;
       // case 'h': argp_state_help (state, stderr, ARGP_HELP_LONG|ARGP_HELP_EXIT_OK);
-    default: return ARGP_ERR_UNKNOWN;
+    default: return (error_t) ARGP_ERR_UNKNOWN;
     }
 
-  return 0;
+  return (error_t)0;
 }
 
 
Index: elfutils-0.188/tests/backtrace.c
===================================================================
--- elfutils-0.188.orig/tests/backtrace.c	2022-11-13 15:20:33.953176394 -0500
+++ elfutils-0.188/tests/backtrace.c	2022-11-13 15:20:33.953176394 -0500
@@ -40,6 +40,10 @@
 #endif
 #include "system.h"
 
+#ifndef PATH_MAX
+#define PATH_MAX 1024
+#endif
+
 #ifndef __linux__
 
 int
