--- elfutils-0.182.orig/tests/dwfl-proc-attach.c
+++ elfutils-0.182/tests/dwfl-proc-attach.c
@@ -107,6 +107,7 @@ main (int argc __attribute__ ((unused)),
   return (threads == 3) ? 0 : -1;
 }
 
+#if 0
 /* HACK. This is a simple workaround for a combination of old glibc
    and valgrind. libdw will try to dlopen libdebuginfod this causes
    some unsuppressable memory leak warnings when the process is
@@ -118,5 +119,6 @@ void *dlopen (void)
 {
   return NULL;
 }
+#endif
 
 #endif /* __linux__ */
